subo/release/version.go

```diff
@@ -12,3 +21,22 @@ var AtmoVersion = "0.4.2"
 
 // SCCTag is the docker tag used for creating new compute core deployments
 var SCCTag = "v0.1.0"
+
+// CheckForLatestVersion returns an error if SuboDotVersion does not match the latest GitHub release or if the check fails
+func CheckForLatestVersion() error {
```

Comment:
This is awesome! Can you add some spacing to your code in this function? We generally leave a blank line after each error check, for example. "Let it breathe!"